qiskit/visualization/pulse_v2/core.py

```diff
@@ -0,0 +1,493 @@
+
+            y0 = offset - (abs(_min_v) + y0_interval)
+
+        # update drawing objects
+        for chan in self.channels:
+            # update channel info to replace scaling factor
+            chan_info = types.ChannelTuple(chan, chan_scale.get(chan, 1.0))
+            for gen in PULSE_STYLE['generator.channel']:
+                for drawing in gen(chan_info):
+                    self._replace_drawing(drawing)
+
+            # update existing drawings
+            for drawing in self.drawings:
+                if drawing.channel == chan:
+                    drawing.visible = chan_visible.get(chan, False)
+                    drawing.offset = chan_offset.get(chan, 0.0)
+                    drawing.scale = chan_scale.get(chan, 1.0)
+
+        # update boundary box
+        self.bbox_bottom = y0 - (PULSE_STYLE['formatter.margin.bottom'] - y0_interval)
+
+        # update axis break
+        if PULSE_STYLE['formatter.control.axis_break']:
+            self._horizontal_axis_break()
+
+    def _ordered_channels(self,
+                          visible_channels: Optional[List[pulse.channels.Channel]] = None) \
+            -> List[pulse.channels.Channel]:
+        """A helper function to create a list of channels to show.
+
+        Args:
+            visible_channels: List of channels to show.
+                If not provided, the default channel list is created from the
+                stylesheet preference.
+
+        Returns:
+            List of ordered channels to show.
+        """
+
+        if visible_channels is None:
+            channels = []
+            for chan in self.channels:
+                # remove acquire
+                if not PULSE_STYLE['formatter.control.show_acquire_channel'] and \
+                        isinstance(chan, pulse.AcquireChannel):
+                    continue
+                # remove empty
+                if not PULSE_STYLE['formatter.control.show_empty_channel'] and \
+                        self.chan_event_table[chan].is_empty():
+                    continue
+                channels.append(chan)
+        else:
+            channels = visible_channels
+
+        # callback function to arrange channels
+        if len(channels) > 1:
+            return PULSE_STYLE['layout.channel'](channels)
+        else:
+            return channels
+
+    def _replace_drawing(self,
+                         drawing: drawing_objects.ElementaryData):
+        """A helper function to add drawing object.
+
+        If the given drawing object exists in the data container,
+        this function just replaces the existing object with the given object
+        instead of adding it to the list.
+
+        Args:
+            drawing: Drawing object to add to the container.
+        """
+        if drawing in self.drawings:
+            ind = self.drawings.index(drawing)
+            self.drawings[ind] = drawing
+        else:
+            self.drawings.append(drawing)
+
+    def _horizontal_axis_break(self):
```

Comment:
This also seems coupled to the current drawing model/format.

Reply:
Right. Currently most of standard plotters don't support axis break. In matplotlib combining two ax with different axis limits is recommended to reproduce axis break (but it is not practical in our case since we may take single ax object from the interface).
Perhaps I need to add functionality that updates drawing data to truncate data points.